site/docs/v1/tech/apis/applications.adoc

```diff
@@ -162,6 +163,122 @@ include::docs/v1/tech/apis/_standard-put-response-codes.adoc[]
 
 include::docs/v1/tech/apis/_application-response-body.adoc[]
 
+== Search for Applications
+
+[NOTE.since]
+====
+This API has been available since 1.44.0
+====
+
+This API is used to search for Applications and may be called using the `GET` or `POST` HTTP methods. Examples of each are provided below. The `POST` method is provided to allow for a richer request object without worrying about exceeding the maximum length of a URL. Calling this API with either the `GET` or `POST` HTTP method will provide the same search results given the same query parameters.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Search for Applications using the given search and sort criteria
+[.endpoint]
+.URI
+--
+[method]#GET# [uri]#/api/application/search``?name=\{name\}&active=\{active\}&tenantId=\{tenantId\}&orderBy={field}&startRow={startRow}``#
+--
+
+==== Request Parameters
+
+[field]#name# [type]#[String]# [optional]#Optional#::
+The string to search for in the Application name. This can contain wildcards using the asterisk or percent characters (* or %).
+
+[field]#active# [type]#[Boolean]# [optional]#Optional#::
+Filters on the Application's `active` status.
+
+[field]#tenantId# [type]#[UUID]# [optional]#Optional#::
+Restricts the results to Applications belonging to the given Tenant.
+
+[field]#numberOfResults# [type]#[Integer]# [optional]#Optional# [default]#defaults to `25`#::
+The number of results to return from the search.
+
+[field]#orderBy# [type]#[String]# [optional]#Optional# [default]#defaults to `insertInstant DESC`#::
+The database column to order the search results on plus the order direction.
++
+The columns you can use for this are:
++
+* `id` - the UUID of the Application
+* `insertInstant` - the link:/docs/v1/tech/reference/data-types#instants[instant] when the Application was created
+* `name` - the Application name
+* `tenant` - the id of the Tenant in which the Application belongs
++
+For example, to order the results by the insert instant in a descending order, the value would be provided as `insertInstant DESC`. The final string is optional can be set to `ASC` or `DESC`.
```

Comment:
Mention that if `ASC` and `DESC` are omitted, it will default to `ASC`.

I would assume most users know that leaving it out in a SQL query will sort in ascending order, but it could be different here.

Reply:
I see now that this comes from another search API. It's probably good.